ic-os/components/setupos-scripts/check-network.sh

```diff
@@ -55,22 +67,14 @@ function get_network_settings() {
         "ip -6 addr show | awk '/^[0-9]+: / {print \$2}' | sed 's/://g' | grep -v '^lo$'" \
         "Failed to get system's network interfaces.")
 
-    if [ -z "${ipv6_capable_interfaces}" ]; then
```

Comment:
Reviewing this, it threw me off that `eval_command_with_retries` checks for an empty result, rather than checking exit codes. Could we add a quick comment to the helper?

Reply:
I just updated eval_command_with_retries to check for exit codes instead! How's that look?

Reply:
Awesome! Maybe we bring back these "empty" checks then too? We should double check that we weren't ignoring any bad exit codes from these commands.

Reply:
Re-added empty checks!